tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/retry/clientserver/RetryClassLevelClientAbortOn.java

```diff
@@ -63,7 +64,7 @@ private void writingService() {
         counterForInvokingWritingService++;
         try {
             Thread.sleep(100);
-            throw new RuntimeException("WritingService failed");
+            throw new TestException("WritingService failed");
```

Comment:
`serviceB` (just above this change in the file), should be updated to abortOn a `TestException` rather than a `RuntimeException`.